tensorflow/python/autograph/operators/py_builtins_test.py

```diff
@@ -198,6 +198,17 @@ def test_print_complex(self):
     finally:
       sys.stdout = sys.__stdout__
 
+  def test_max(self):
+    self.assertEqual(py_builtins.max_([1, 3, 2]),3)
+    self.assertEqual(py_builtins.max_(0,1),1)
+
+  def test_max_tensor(self):
```

Comment:
A test with nonscalars, e.g. `constant([[[1]]])`, as well as higher-rank (with `assertRaises`) would be nice too.

Reply:
Yes locally I've prepared something like that before the cache was invalidated after a reboot.